Jenkinsfile

```diff
@@ -214,20 +218,9 @@ void stageAutomaticRelease() {
         }
 
         stage('Regenerate resources for release') {
-            new Docker(this)
-                    .image("golang:${goVersion}")
-                    .mountJenkinsUser()
-                    .inside("--volume ${WORKSPACE}:/go/src/${project} -w /go/src/${project}")
-                            {
-                                make 'k8s-create-temporary-resource'
-                            }
-        }
-
-        stage('Push to Registry') {
-            GString targetOperatorResourceYaml = "target/${repositoryName}_${controllerVersion}.yaml"
-
-            DoguRegistry registry = new DoguRegistry(this)
-            registry.pushK8sYaml(targetOperatorResourceYaml, repositoryName, "k8s", "${controllerVersion}")
+            make 'crd-helm-generate'
```

Comment:
You don't have to regenerate because the package targets will do this.

Reply:
Right right, but what about the artifact archiving? Should we then join the stage `Regenerate resources for release` with `Push Helm chart to Harbor` because the of the same target chain?